### PR TITLE
DOCSP-41973 Read Landing Page

### PR DESCRIPTION
**Pull request by lindseymoore**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-41973>
Staging - <https://deploy-preview-148--docs-php-library.netlify.app/read/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read.txt [hunk 1]
```diff
@@ -16,3 +30,134 @@ Read Data from MongoDB
...
+{+driver-short+} methods for retrieving documents.
+
+.. tip::
+
+   To learn more about any of the methods shown on this page, see the link
+   provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <php-read-sample>` or your own application.
+Be sure to replace all placeholders, such as ``<connection string>``, with
```
Comment (mcmorisi) [inline source/read.txt:47]: The `<connection string>` placeholder doesn't actually appear in the code sample. Suggest either adding it in there or changing this for consistency.

**Review by mcmorisi — APPROVED** [review]
LGTM with one fix!

File: source/includes/usage-examples/read-code-examples.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    if ( ! $changeStream->valid()) {
+        continue;
+    }
+    $event = $changeStream->current();
+    echo toJSON($event) . PHP_EOL;
+
+    if ($event['operationType'] === 'invalidate') {
+        break;
+    }
+}
```
Comment (GromNaN) [inline source/includes/usage-examples/read-code-examples.php:66]: `do...while` would be more appropriate.

```suggestion
$changeStream->rewind();

do {
    $changeStream->next();

    if ($changeStream->valid()) {
        $event = $changeStream->current();
        echo toJSON($event) . PHP_EOL;
    }
} while ($event['operationType'] !== 'invalidate');
```

Have you looked at our [changestream example](https://github.com/mongodb/mongo-php-library/blob/v1.x/examples/changestream.php)?

Comment (lindseymoore) [reply source/includes/usage-examples/read-code-examples.php:66]: For consistency with examples on the already approved [Monitor Data Changes page](https://deploy-preview-148--docs-php-library.netlify.app/read/change-streams/), I'll keep as is here. I can look into revising the examples in the future. Thanks!

**Pull request merged**